docs/docs/articles/testkube-cli-docker.md

```diff
@@ -45,7 +45,7 @@ make docker-build-cli DOCKER_BUILDX_CACHE_FROM=type=registry,ref=docker.io/kubes
 Once you have the image, run the following command pointing to the Testkube demo:
 
 ```bash
-docker run kubeshop/testkube-cli:latest version --namespace testkube --api-uri https://demo.testkube.io/results --client direct
+docker run kubeshop/testkube-cli:latest testkube version --namespace testkube --api-uri https://demo.testkube.io/results --client direct
```

Comment:
It won't work - every argument passed here will be executed as separate command, due to:
```sh
for cmd in "$@"; do eval "$cmd"; done
```